README.md

```diff
@@ -17,31 +17,28 @@ You can also download the required k6 binary from the [releases page](https://gi
 # Example
 
 ```javascript
-import { KubernetesClient } from './src/chaos.js';
+import { Kubernetes } from 'k6/x/kubernetes';
+import { KubernetesChaosClient } from '../src/chaos.js';
 
 export default function () {
-  // We instantiate a new *chaotic* Kubernetes client.
-  const kubernetes = new KubernetesClient()
 
-  // We run KillRandomPod to randomly kill a pod in the specified namespace.
-  const killed = kubernetes.killRandomPod('operations')
-  console.log(`Killed pod: ${killed}`);
+  const k8sClient = new Kubernetes()
 
-  // And then, we run DeleteRandomNamespace to delete a random namespace.
-  const deleted = kubernetes.deleteRandomNamespace()
-  console.log(`Deleted namespace: ${deleted}`);
+  // randomly kill a job of the given namespace.
+  KubernetesChaosClient.killRandomJob(k8sClient, jobs[0].namespace);
```

Comment:
Instead of moving the k8sClient around, how do you feel about something like this:
```javascript
const k8sClient = new Kubernetes()
const chaosClient = new ChaosClient().withK8sClient(k8sClient)
// another option is: const chaosClient = new ChaosClient(k8sClient)

// Then, you can use the wrappers/helpers
chaosClient.k8s.killRandomJob(jobs[0].namespace);
// This approach, could work with other kind of "targets"
chaosClient.ssh.whateverSomething();
```

Reply:
@dgzlopes 

Are you ok with?

```javascript
import { Kubernetes } from 'k6/x/kubernetes';
import { KubernetesChaos } from '../src/kubernetes.js';


export default function () {
  const k8sClient = new Kubernetes()
  const k8sChaos = new KubernetesChaos(k8sClient)
```

If we are gonna expose `k8s, ssh, docker....`. Perhaps, better to have a dedicated class for each client.

Reply:
That's nice! Yeah, works for me. Agree, having a dedicated class it's going to be easier to maintain and reason/about.